inspect/network.go

```diff
@@ -65,7 +65,7 @@ func (ni *NetworkInspector) Inspect(ctx context.Context, inspectionCfg Inspectio
 }
 
 // speedTest returns download speed, upload speed and error.
-func speedTest() (float64, float64, error) {
+func _() (float64, float64, error) {
```

Comment:
Does the linter error go away if we rename to `SpeedTest` instead? That could be a little more useful.